backend/src/v5/schemas/tickets/tickets.groups.js

```diff
@@ -25,19 +25,39 @@ const Groups = {};
 
 const uuidString = Yup.string().transform((val, orgVal) => UUIDToString(orgVal));
 
+const objectEntryValidator = Yup.object().shape({
+	container: Yup.string().test('Container id', 'Container ID must be an UUID string', isUUIDString).required(),
+	_ids: Yup.array().of(types.id).min(1),
+	ifc_guids: Yup.array().of(Yup.string().length(22)).min(1),
+	revit_ids: Yup.array().of(Yup.number()).min(1),
+}).test(
+	'Object item check',
+	'Can only contain either _ids or ifc_guids or revit_ids',
+	/* eslint-disable no-underscore-dangle */
+	(value) => (value._ids && !(value.ifc_guids || value.revit_ids))
```

Comment:
Unless i'm missing something, this logic would allow me to have no ids? I think `{container: "<some uuid>"}` would pass here.

If this is really true then you're missing a unit test...